aten/src/THC/THCAtomics.cuh

```diff
@@ -103,6 +103,24 @@ static inline __device__ void atomicAdd(bool *address, bool val) {
   *address = address && val;
 }
 
+#ifdef __HIP_PLATFORM_HCC__
```

Comment:
I don't see any ROCm specific api in the code, so I think it should work on cuda as well?

Reply:
AFAIK there is no bf16 support for CUDA yet?

Reply:
`at::BFloat16` should work on CUDA

Can you try removing the ifdef and see if cuda CI jobs complain?

Reply:
There is basic support. Please see [PR](https://github.com/pytorch/pytorch/pull/27259)